Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM golang:1.17.8 as build
+RUN mkdir -p /go/go.etcd.io/etcd
+WORKDIR /go/go.etcd.io/etcd
+ADD go.mod go.mod
```

Comment:
In general COPY would be better: https://github.com/hadolint/hadolint/wiki/DL3020